File: python/S01-elemental-actions/E05-frog-jumps.py
def main():
	print("\n\033[35m[========= FROG JUMPS =========]\033[0m\n")

	joseDistance: int = int(input("Enter the distance in centimeters that José has to jump: "))
	pedroDistance: int = int(input("Enter the distance in centimeters that Pedro has to jump: "))

	joseJumps: int = int(input("Enter the number of jumps that José can make: "))

	if (joseDistance * joseJumps % pedroDistance == 0):
		print("José can jump the distance in one jump.")
	else:
		print("José can't jump the distance in one jump.")


if __name__ == "__main__":
	main()

File: c++/S01-elemental-actions/E05-frog-jumps.cpp
#include <iostream>

int main(int argc, char *argv[]) {
	std::cout << "\n\e[0;35m[========= FROG JUMPS =========]\e[0m\n\n";

	int joseDistance, pedroDistance, joseJumps;

	std::cout << "Enter the distance Jose can jump: ";
	std::cin >> joseDistance;
	std::cout << "Enter the times Jose will jump: ";
	std::cin >> joseJumps;
	std::cout << "Enter the distance Pedro can jump: ";
	std::cin >> pedroDistance;
	std::cout << '\n';

	if ( (joseDistance * pedroDistance) <= (joseDistance * joseJumps) ){
		std::cout << "The frogs will meet at the same point before Jose finishes all his jumps." << '\n';
	} else{
		std::cout << "The frogs will not meet at some point before Jose finishes all his jumps." << '\n';
	}

	return 0;
}
